.github/CODEOWNERS

```diff
@@ -1540,10 +1541,9 @@ sdk/ai/ai-inference-rest @glharper @dargilco
 
 /eng/tools/dependency-testing @KarishmaGhiya @jeremymeng
 
-/rush.json @mikeharder @ckairen @jeremymeng @Azure/azure-sdk-for-js-core
+/rush.json @mikeharder @ckairen @jeremymeng @Azure/azure-sdk-js-dev
 /tsconfig.json @mikeharder @ckairen @jeremymeng @deyaaeldeen
-/**/tsconfig.json @jeremymeng @deyaaeldeen
-/**/tsconfig.strict.json @deyaaeldeen
+/**/tsconfig.json @jeremymeng @deyaaeldeen @Azure/azure-sdk-for-js-core
```

Comment:
it seems this still blocked us?
https://github.com/Azure/azure-sdk-for-js/pull/30341

![image](https://github.com/user-attachments/assets/4857d964-fa9a-4bd2-b9af-2743092409bf)

Reply:
oh yeah I will need a entry for mgmt path I guess, similar to the api.md one.